Libraries/PushNotificationIOS/RCTPushNotificationManager.m

```diff
@@ -346,7 +345,6 @@ - (void)handleLocalNotificationReceived:(NSNotification *)notification
                                                  categories:categories];
 
     [RCTSharedApplication() registerUserNotificationSettings:settings];
-    [RCTSharedApplication() registerForRemoteNotifications];
```

Comment:
Do we do this anywhere else? I imagine we still want to register for remote notifications *somewhere*, right?